appyters/Reactome_Harmonizome_ETL/Reactome.ipynb

```diff
@@ -466,15 +457,15 @@
    "outputs": [],
    "source": [
     "edge_list = uf.edge_list(binary_matrix)\n",
-    "uf.save_data(edge_list, path, output_name + '_edge_list', \n",
-    "        ext='tsv', compression='gzip')"
+    "uf.save_data(edge_list, path, output_name + '_edge_list',\n",
+    "             ext='tsv', compression='gzip')"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "# Create Downloadable Save File"
+    "**Create Turtle RDF**"
```

Comment:
It would be ideal to use `#` notation as to improve the table of contents, though it should be some deeper level (i.e. `###`). The same can be said to the other headers which were converted to bold face.